src/client/src/consumer.rs

```diff
@@ -349,3 +389,18 @@ impl ConsumerConfig {
         self
     }
 }
+
```

Comment:
may mark as hidden?

Reply:
Are you talking about the Record type?

Reply:
Yes

Reply:
I think that although this Record type has `pub` in it, the `consumer` module is not pub, so any clients will not be able to import the Record name. I think this is the behavior we want.

Reply:
fair enough